dokku

```diff
@@ -211,6 +222,23 @@ case "$1" in
     pluginhook update
     ;;
 
+  plugins:disable)
+    PLUGIN="$2"
+    [[ -e $PLUGIN_CORE_PATH/$PLUGIN ]] && echo "Cannot disable a core plugin" && exit 1
+    [[ -e $PLUGIN_DISABLED_PATH/$PLUGIN ]] && echo "Plugin already disabled" && exit 1
+    [[ ! -e $PLUGIN_PATH/$PLUGIN ]] && echo "Plugin does not exist" && exit 1
+    mv "$PLUGIN_PATH/$PLUGIN" "$PLUGIN_DISABLED_PATH"
+    echo "Plugin $PLUGIN disabled"
```

Comment:
Yeah but if we're going to have two enabled places then we'll need to manage the plugin path ourselves somehow.